soap-attachments/2.0/_index.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: "Jakarta SOAP Attachments 2.0 (milestone)"
-date: 2020-06-05
+title: "Jakarta SOAP Attachments 2.0"
+date: 2020-09-30
```

Comment:
@lukasj Just a comment... The ballot is now closed for this Specification.  So, when this PR gets merged, this date field will determine when to make the web page go live.  You might want to consider changing this to 2020-09-16 to allow for immediate consumption once the merge is complete.  If you don't see this comment before I'm ready to merge, then I'll make the update when I do the ballot update.  Thanks.

Reply:
What's the policy here, because both Authorization, Mail and others did not change the date in the top to September, should we do that?

Reply:
@kwsutter my assumption was that the spec is about to go live once the release of the api and impl is approved by pmc etc which is going to need another few weeks (actually the api/impl cannot be release before Oct 7, since release review is to conclude on Oct 7 unless I'm missing something) If you don't mind having broken links for that long, feel free to change the date

Reply:
I realize the Release Reviews are necessary, but we still don't have that process running smoothly yet.  As Werner pointed out, we already had a couple of Spec PRs get approved without a formal Release Review.  I will go ahead and merge this as-is, but I will probably change this when I submit the ballot update.  In my mind, if the ballot is approved, then the Release Review is a formality that will eventually even out.  Eventual consistency... :-)

Reply:
@kwsutter https://www.eclipse.org/projects/handbook/#specification-project-reviews is usually my source. Whatever gets decided should be reflected in the handbook too